src/ussd_clinic_rapidpro.js

```diff
@@ -35,6 +36,20 @@ go.app = function() {
                 self.im.config.services.whatsapp.base_url,
                 self.im.config.services.whatsapp.token
             );
+
+            self.env = self.im.config.env;
+            self.metric_prefix = [self.env, self.im.config.name].join('.');
+            
+            self.im.on('state:enter', function(e) {
+                return self.im.metrics.fire.sum('enter.' + e.state.name, 1);
+            });
+
+            var mh = new MetricsHelper(self.im);
+            mh
+                // Total sum of users for each state for app
+                // This adds <env>.ussd_clinic_rapidpro.sum.users_per_state.transient 'sum' metric
```

Comment:
I don't think that this adds a `<env>.ussd_clinic_rapidpro.sum.users_per_state.transient` metric, I think it adds a `<env>.ussd_clinic_rapidpro.sum.unique_users` `last` metric, and a `<env>.ussd_clinic_rapidpro.sum.unique_users.transient` `sum` metric